main.js

```diff
@@ -28,7 +28,7 @@ function createWindow () {
 
   mainWindow.setAlwaysOnTop(true, "floating");
   mainWindow.setVisibleOnAllWorkspaces(true);
-  mainWindow.setFullScreenable(false);
+  mainWindow.setFullScreenable(true);
```

Comment:
This should be set only when keys are pressed for full screen as this will disable always-on-top over full-screen apps. Similarly, it should enable frame for the full screen so that it can be controlled via mouse/cursor
Also, add a listener for "Esc" when the app is in focus to get out of full screen.
Then there will need to listen for the event when app leaves the full screen, there we will again disable fullscreen config, and remove the frame.